src/typesetting/PSandAI.md

```diff
@@ -0,0 +1,30 @@
+# Masking using AI and PS
+# Preface
+
+So before we get to know on how to work with AI,
+I would like to give a small warning.
```

Comment:
Using "I" in a guide that is maintained by a community is usually a bad idea, unless the author of a page is clearly denoted, because the author may change across pages and sometimes even within the same page. 

Please either add something that allows the reader to identify the author as a single entity and allows them to make the connection to someone in particular (read: you), or rephrase everything to be more general, e.g. by using passive forms like "Before starting, a small warning is warranted." or use "we" to refer to the contributors collectively. Choose whatever you prefer since I don't have a strong opinion on it yet, but know that others may contribute to this page as they and the reviewer see fit.

Reply:
I have mentioned myself at the beginning and made clear it is me and I would like to keep it that way through out my guides (Mostly I will write TS/Timing stuff in future)